projects/sbb-esta/angular-public/autocomplete/src/autocomplete-hint/autocomplete-hint.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, ElementRef, Input, OnInit } from '@angular/core';
+
+/**
+ * Autocomplete-hint IDs need to be unique across components, so this counter exists outside of
+ * the component definition.
+ */
+let nextId = 0;
+
+@Component({
+  selector: 'sbb-autocomplete-hint',
+  templateUrl: './autocomplete-hint.component.html',
+  styleUrls: ['./autocomplete-hint.component.scss']
+})
+export class AutocompleteHintComponent implements OnInit {
+  /**
+   * Takes classes set on the host sbb-autocomplete-hint element and applies them to the panel
+   * inside the overlay container to allow for easy styling.
+   */
+  @Input('class')
```

Comment:
This (and _classList) is not necessary, since this element content is not going to be extracted.